indium-chrome.el

```diff
@@ -113,6 +113,21 @@ Try a maximum of NUM-TRIES."
           (port (read-from-minibuffer "Port: " (number-to-string indium-chrome-port))))
       (indium-chrome--get-tabs-data host port #'indium-chrome--connect-to-tab))))
 
+;;;###autoload
+(defun indium-connect-to-local-chrome-to-url (url)
+  "Connect to a local Chrome instance to the given tab URL."
+  (indium-chrome--connect-to-chrome-to-url "localhost" "9229" url))
+
+;;;###autoload
+(defun indium-connect-to-chrome-to-url (host port url)
+  "Connect to a Chrome instance with HOST:PORT to a given tab URL."
+  (indium-chrome--connect-to-chrome-to-url host port url))
+
+(defun indium-chrome--connect-to-chrome-to-url (host port url)
```

Comment:
Could this function be inlined?

Reply:
I'll leave for now to make the tests pass. Let me know if there is a better way.